index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compactible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./styles.css" />
+    </head>
+    <body>
+        <div class="top">
+        <header class="header">
+            <h2 class="logo"><a class="logo" href="#home">PETER</a></h2>
+    <nav class="navbar">
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
+                    fill="#acb7c3" />
+            </svg>              
+        
+        <ul class="navigation">
+            <li class="portfolio"><a href="#home">portfolio</a></li>
+            <li class="abt"><a href="#about">About</a></li>
+            <li class="Contact"><a href="#contact">Contact</a></li>
+            <i class='fas'>&#xf199;</i>
+        </ul>             
+    </nav>
+    </header>
```

Comment:
- [ ] Kindly stick to the Figma design your `navbar` is not the same as the Figma design so kindly fix these issues. Check below :point_down:  the comparison between your design and Figma design :+1: 

**Your Design**
![Screenshot from 2023-03-05 02-57-33](https://user-images.githubusercontent.com/45226848/222930599-75fe22df-8ae4-4406-af2c-72af65aa7e61.png)

**Figma Design**
![Screenshot from 2023-03-05 03-02-43](https://user-images.githubusercontent.com/45226848/222930624-ff2fa446-b386-4aae-b0e3-4d180d3cf620.png)

- [ ] If you check your design there is no spacing at the start and at the end of the `navbar` and also your icon is not matching with Figma design icon kindly fix this issue. :+1: